pkg/kubectl/cmd/create_service.go

```diff
@@ -51,20 +51,22 @@ var (
 
 	serviceClusterIPExample = templates.Examples(i18n.T(`
     # Create a new ClusterIP service named my-cs
-    kubectl create service clusterip my-cs --tcp=5678:8080
+    kubectl create service clusterip my-cs --ports=tcp:5678:8080
```

Comment:
Why not add a `--udp` flag?

Reply:
i think one generic flag is better than specific flag to expand the future function extension and also tcp/udp/ belong to the same class of tcp/ip protocol. If we support both --tcp and --udp flag we also need to think about the  mutual exclusion.

Reply:
i think use `--protocol` and `--port` would be better

Reply:
> i think use --protocol and --port would be better

that makes it difficult to expose multiple ports on different protocols

Reply:
I vote for `--udp`